### ISSUE TITLE
Loss exploded on some datasets

### ISSUE THREAD

**Issue by XiaooaiX** [Rayhane-mamah/Tacotron-2#345]
Dear all,
I have a problem these days about this repo that when I train this taco2 model on some dataset, the loss appears to get exploded after a number of steps.  Just like this:

![capture](https://user-images.githubusercontent.com/26372106/53948891-d4a80800-4103-11e9-94a7-509dc0ca95ab.PNG)

Anyone knows how to fix it?

Thanks!

**Comment by Bobb10** [comment]
@XiaooaiX. Can you share the solution for this issue? I have the same problem with a custom dataset, its loss explodes after 35K of wavenet training steps.

**Comment by dalvlv** [comment]
@XiaooaiX  Hi friend, did solve the 'Loss Exploded' problem?  When I trained Mandarin dataset, I met the same problem. How did you solve it?

**Comment by dalvlv** [comment]
@Bobb10. Loss value of my train turns to negatigve quickly at start, do you know the reason？ And also after 40K steps, the loss exploded.

**Comment by Bobb10** [comment]
@dalvlv. I haven't found a reason why the loss explodes while training WaveNet. It explodes sort of randomly after several steps (not just 35K, but also further in the training stage). Would be more than happy to receive an answer if someone knows why this is happening and how to solve this.
